sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/PhoneNumberClient.java

```diff
@@ -539,4 +539,19 @@ public void purchaseSearch(String searchId) {
         CreateSearchOptions options, Duration pollInterval) {
         return phoneNumberAsyncClient.beginCreateSearch(options, pollInterval).getSyncPoller();
     }
+
+    /**
+     * Creates a release for the given phone numbers.
+     * This function returns a Long Running Operation poller.
+     * 
+     * @param phoneNumbers A list of {@link PhoneNumber} with the desired numbers to release
+     * @param pollInterval The time our long running operation will keep on polling 
+     * until it gets a result from the server
+     * @return A {@link SyncPoller} object with the release entity
+     */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
```

Comment:
The return type is a poller, shouldn't it be single.. I am not sure why it should be a COLLECTION